3_javascript/javascript1/week3/script3.js

```diff
@@ -0,0 +1,135 @@
+// Remove the item in names that is equal to nameToRemove
+const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+  ];
+  const nameToRemove = "Ahmad";
+  
+let remainingNames = names.filter(name => name !==nameToRemove);  
```

Comment:
In this case, your variable is not changing, so consider using `const` instead of `let`.

Reply:
Changes made as recommended!